fedimint-core/src/bitcoin_migration.rs

```diff
@@ -155,6 +183,12 @@ pub fn bitcoin29_to_bitcoin30_sha256_hash(
     bitcoin30::hashes::sha256::Hash::from_byte_array(hash.into_inner())
 }
 
+pub fn bitcoin30_to_bitcoin29_sha256_hash(
+    hash: bitcoin30::hashes::sha256::Hash,
+) -> bitcoin::hashes::sha256::Hash {
+    bitcoin::hashes::sha256::Hash::from_str(&hash.to_string()).unwrap()
```

Comment:
Why always strings and not bytes :sob: there probably is some `into_inner`/`to_bytes` function.

Reply:
Ope 🙊 my bad! Fixed.